### PR TITLE
Get the time from dlt messages for DTLBroker

### PR DESCRIPTION
**Pull request by yen3**
There is an error between dlt message time and system time. For getting precise timestamp in message parsing, the patch adds a share memory to pass the timestamp from DLTMessageHandler to DLTBroker.

Please refer `dlt_broker.py` to get more implementation details and discussions.

### CODE REVIEW

File: dlt/dlt_broker.py [hunk 1]
```diff
@@ -85,21 +93,26 @@ def remove_context(self, context_queue):
 
     def stop(self):
         """Stop the broker"""
-        logger.info("Stopping DLTContextHandler and DLTMessageHandler")
+        logger.info("Stopping DLTContextHandler, DLTTimeHandler and DLTMessageHandler")
```
Comment (khiz678) [inline dlt/dlt_broker.py:96]: There is no more DLTTimeHandler, please remove here.

File: tests/dlt_broker_time_tests.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2021. BMW Car IT GmbH. All rights reserved.
+"""Test DLTTimeHandler"""
```
Comment (khiz678) [inline tests/dlt_broker_time_tests.py:2]: No more DLTTimeHandler....

**Review by khiz678 — CHANGES_REQUESTED** [review]

File: dlt/dlt_broker.py (hunk 1 above)
Comment (yen3) [reply dlt/dlt_broker.py:96]: done

File: tests/dlt_broker_time_tests.py (hunk 2 above)
Comment (yen3) [reply tests/dlt_broker_time_tests.py:2]: done

**Review by khiz678 — APPROVED** [review]

**Pull request merged**